ingeniamotion/motion_controller.py

```diff
@@ -146,6 +153,8 @@ def info(self) -> Information:
     @property
     def fsoe(self) -> "FSoEMaster":
         """Instance of :class:`~ingeniamotion.fsoe.FSoEMaster` class"""
+        if not FSOE_MASTER_INSTALLED:
+            raise NotImplementedError("The FSoE module is not available.")
```

Comment:
I would indicate that the package must be installed with the extra specified:
`pip install ingeniamotion[FSoE]`

Reply:
d23549a